Set-SslSecurity.ps1

```diff
@@ -98,6 +98,11 @@ process {
     } until($response.StatusCode -eq '200')
     Write-Host "Nexus is ready!"
 
+    # Update Repository URI
+    choco source remove --name="'ChocolateyInternal'"
+    $RepositoryUrl = "https://${SubjectWithoutCn}:8443/repository/ChocolateyInternal/"
+    choco source add --name="'ChocolateyInternal'" --source="'$RepositoryUrl'" --allow-self-service --priority=1
```

Comment:
Why `--allow-self-service` here?

Reply:
I agree it's not necessary. If you want to add it as a suggestion, I will happily accept the commit. 🙂